umbraco-cloud/getting-started/flexible-environments.md

```diff
@@ -30,16 +28,28 @@ This feature enables:
 * **Easier hotfixes and feature releases**: Urgent fixes can be deployed directly without being blocked by unfinished work in other environments.
 * **Improved Content Management**: Content editors can create, test, and validate content changes without depending on a specific environment.
 
+## How it works
+
+* A flexible environment is added and therefor connected to a single mainline environment.
+* Developers develop and build features in the flexible environment without affecting the mainline environment.
+* Once a feature is complete, it can be merged back into the mainline environment and become part of regular deployment flow.
+* When there are changes made on the mainline environment, these need to be pulled into the flexible environment before changes can be pushed.
+
+Learn more about the deployment process in the [Deployments](../deployment/README.md) section.
+
 ## Project Prerequisites
 
 Before you can add a Flexible environment to your project, the following prerequisites must be met:
 
-* Uses Umbraco Version 10, 13, 14 or greater.
+* Uses Umbraco Version 10, 13, 14 or greater.
 * Uses Deploy Version Greater than 10.4.1, 13.3.0, 14.2.0 or greater.
 
-{% hint style="info" %}
-Flexible Environments are not available on Heartcore projects.
-{% endhint %}
+### Limitations
+
+* Only one flexible environment is available.
```

Comment:
Here, **one** is spelled out, and on line 20, it's in numbers. I prefer the spelled-out version since it's more formal. But you can choose either one and maintain consistency in the article.